src/components/EmailNotifications/EmailNotificationsForm.js

```diff
@@ -225,7 +225,7 @@ const LegalTermsAndPolicy = React.memo(function LegalTermsAndPolicy({
           `}
         >
           By continuing with your email, you agree to Aragon Protocol's{' '}
-          <Link href="https://anj.aragon.org/legal/terms-general.pdf">
+          <Link href="https://ant.aragon.org/legal/terms-general.pdf">
```

Comment:
Let's maybe leave a TODO for updating or checking if the url will exist ?

Reply:
i already wrote this here in the comment waiting for brett or somebody to tell us if is going to exist or not hehe

Reply:
Ah sorry about that 😄